addon/components/behave/scrollable.hbs

```diff
@@ -0,0 +1,30 @@
+<div
+  class="
+    Behave-Scrollable
+    -{{this.direction}}
+    {{if this.isEnabled "-enabled"}}
+    {{if this.isAtStart "-start"}}
+    {{if this.isAtEnd "-end"}}
+  "
+
+  ...attributes
+>
+  <div class="Behave-Scrollable-Start {{this.startClass}}">
+    {{yield to="start"}}
+  </div>
+
+  <div class="Behave-Scrollable-End {{this.endClass}}">
+    {{yield to="end"}}
+  </div>
+
+  <div
+    class="Behave-Scrollable-Content {{this.contentClass}}"
+
+    {{(if this.isEnabled (modifier "element-query" sizes=false onResize=trackSize))}}
```

Comment:
I'll use this:

[ember-resize-observer-modifier](https://github.com/alexlafroscia/ember-resize-observer-modifier)

because it relies on the same `ember-resize-observer-service` as `ember-element-query` (will reuse one `ResizeObserver` instance).

Reply:
Wait, it doesn't! It's `ember-on-resize-modifier` that does. 👍

Reply:
Done.